src/DotNetty.Transport/Channels/IChannelId.cs

```diff
@@ -0,0 +1,12 @@
+﻿
+using System;
+
+namespace DotNetty.Transport.Channels
+{
+    public interface IChannelId : IComparable<IChannelId>
+    {
+        string AsShortText { get; }
```

Comment:
Better to keep this (and `AsLongText`) as methods indicating that considerable amount of work is performed when it's accessed.